rusoto/credential_service_mock/run-and-test.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+cargo build; cargo run &
+
+cd ../credential ; sleep 5 ; cargo test -- --ignored ; killall credential_service_mock
```

Comment:
The `sleep` call is to let the `warp` server come up. We could be smarter about waiting than a hardcoded wait like this.